docs/ref/contrib/postgres/fields.txt

```diff
@@ -920,3 +920,28 @@ define :class:`~django.contrib.postgres.constraints.ExclusionConstraint`. See
 docs/current/rangetypes.html#RANGETYPES-INCLUSIVITY>`_.
 
 .. _different bounds: https://www.postgresql.org/docs/current/rangetypes.html#RANGETYPES-IO
+
+``Serial`` fields
+=================
+
+.. versionadded:: 5.1
+
+.. class:: SerialField(**options)
+
+    A field for storing an automatically incremented sequence of integers.
+
+    It's similar to :class:`AutoField <django.db.models.AutoField>`, but it
+    doesn't have to be a primary key.
```

Comment:
```suggestion
    An automatically incrementing :class:`IntegerField`, much like :class:`AutoField`
    except that it does not have to store a primary key.
```
Maybe

Reply:
`it does not have to ***be*** a primary key.` sounds more natural to me, but I'm not a native English speaker.

Reply:
Yes I like it 👍